src/Tool-DependencyAnalyser/DAPackageDependency.class.st

```diff
@@ -22,8 +22,9 @@ DAPackageDependency class >> from: aSourcePackage to: aDestinationPackage [
 	^ self new initializeFrom: aSourcePackage to: aDestinationPackage
 ]
 
-{ #category : 'printing' }
+{ #category : 'displaying' }
```

Comment:
Personally I find that printing and displaying are kind of the same. I'm not sure I'd like two protocols for a usage that is so close. When browsing code I'd prefer to have both grouped and not have a thousand protocols with really few methods in each

Reply:
When they would be the same - why make a distinction in selector then? Why call one set of variants "displayXXX", while other variants "printXXX" and a third one "storeXXX". Because they serve a different purpose and this should be reflected in the protocol name too.

You can call me one of those grumpy old Smalltalkers (TM) around - but this is useful and more explicit.
To my knowledge in most of the different Smalltalk systems (commercial and non-commercial) such a distinction in name and protocol is made. And it made perfectly sense - especially when you work more extensively with UI where displaying an object is used more often on the "external" (exposed) user interface while printing is used more for "internal" printable representations in inspectors and dev tools like the debugger.

See for instance Discord posts like this:
<img width="529" alt="image" src="https://github.com/pharo-project/pharo/assets/5980033/008ce0de-c3cb-4047-8ad1-553776a204aa">